docs/guide/contributing.md

```diff
@@ -214,6 +214,25 @@ We cannot control the `html` base font size and therefore these relative units a
 
 TODO: wip
 
+#### Attributes
+
+Sometimes you will need to generate a random string to be used as value for various attributes (e.g. accessibility-related attributes like `id`, `for`, `aria-labelledby`, etc.). To generate a random id you can use a `useUniqueId` composable:
```

Comment:
```suggestion
Sometimes you will need to generate a random string to be used as value for various attributes (e.g. accessibility-related attributes like `id`, `for`, `aria-labelledby`, etc.). 

To generate a random id so that it is safe for SSR, you **must** use the `useUniqueId` composable in your component:
```